tests/components/wrappings/wrap-events.test.js

```diff
@@ -198,3 +198,22 @@ test('should support listener object with handleEvent method', async () => {
 
   expect(handler.handleEvent).toHaveBeenCalledTimes(1)
 })
+
+test('AEL on window should call through to AEL on EventTarget', async () => {
```

Comment:
Avoid abbreviations please.

```suggestion
test('calling addEventListener on window should call through to addEventListener on EventTarget', async () => {
```